gconfig/DecayConfigPy6.C

```diff
@@ -5,6 +5,9 @@ void DecayConfig() {
 	// specific types of decays defined below.
 
 	// Access the external decayer singleton and initialize it
+	if (TString(gSystem->DynamicPathName("libtpythia6.so", kTRUE)) != TString("")){
```

Comment:
I don't think this is needed, if the ROOT dictionaries are in the correct place.

Reply:
You are right, 

I believe the issue with this and the similar comment is that I should use the same library name as the ROOT old TPythia6 library. libEGPythia6.so, instead of libtpythia6.so. 

I will do it and remove all these redundant if conditions.

Reply:
Indeed, using the same library name seems like a good idea.

Thanks for the update!